crates/bevy_render/src/render_phase/draw_state.rs

```diff
@@ -270,16 +274,33 @@ impl<'a> TrackedRenderPass<'a> {
             .set_viewport(x, y, width, height, min_depth, max_depth)
     }
 
+    /// Insert a single debug marker.
+    ///
+    /// This is a GPU debugging feature. This has no effect on the rendering itself.
```

Comment:
I would like to see a simple doc example here, or more breadcrumbs to how this might be used.

Reply:
Looking a bit more, I think that the debugging advice should actually be described in more depth as part of the `TrackedRenderPass` struct that owns these methods. It's not essential for this PR though.

Reply:
<img width="954" alt="debug" src="https://user-images.githubusercontent.com/6532395/147504314-fccce4d1-2fd4-48d8-986b-6efb58ccbcda.png">